### PR TITLE
Modal

### PR DESCRIPTION
**Pull request by chanchal-panpaliya**
temp project live - https://solutionground.netlify.app/

### CODE REVIEW

File: modal/utility/utility-icon-size.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    font-size: 2rem !important;
+  }
+  
+  .icon-size-small{
+    font-size: 1rem !important;
+  }
+  
+  .icon-size-extrasmall{
+    font-size: 0.5rem !important;
+  }
```
Comment (Mohasin-Haque) [inline modal/utility/utility-icon-size.css]: avoid using important

File: modal/utility/utility-flex.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    flex-wrap: wrap;
+}
+
+.flex-row{
+ flex-direction: row;
+ flex-wrap: wrap;
+ min-height: auto;
+ max-width: auto;
+ /* min-height: 100%;
+ max-width: 100%; */
```
Comment (Mohasin-Haque) [inline modal/utility/utility-flex.css]: nedds to remoed

**Review by Mohasin-Haque — COMMENTED** [review]
Your code is god but it needs to be formatted and clean. It will be good if you remove the whitespaces and add meaningful comments!!

File: modal/utility/utility-icon-size.css (hunk 1 above)
Comment (chanchal-panpaliya) [reply modal/utility/utility-icon-size.css]: I can't change the size of an icon (material-ui) without  using - "!important" .

File: modal/utility/utility-flex.css (hunk 2 above)
Comment (chanchal-panpaliya) [reply modal/utility/utility-flex.css]: done

**Comment by poojakelkar** [comment]
Remove Important, Otherwise looks Good to merge.

File: modal/Modal/Modal.html [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+        <li> <a href="#simple-link-sidebar" class="color-black sidebar-onhover-primary"> Simple Modal </a></li>
+        <li> <a href="#actionmodal-link-sidebar" class="color-black sidebar-onhover-primary"> Action Modal </a> </li>
+    </ul>
+    </div>
+</aside>
+<!-- container closed-->  
+</div>
+<script src="Modal.js"> </script>
+</body>
+</html>
```
Comment (darshandamre) [inline modal/Modal/Modal.html]: you can format this piece of code

**Review by darshandamre — COMMENTED** [review]
one more suggestion is you can add some padding to the cancel and submit button in action modal

File: modal/Modal/Modal.html (hunk 3 above)
Comment (chanchal-panpaliya) [reply modal/Modal/Modal.html]: modalimg

**Pull request merged**